src/os/bluestore/KernelDevice.cc

```diff
@@ -456,12 +456,16 @@ int KernelDevice::aio_zero(
   assert(off + len <= size);
 
   bufferlist bl;
+  bufferlist t(len);
```

Comment:
* no len arg to the bufferlist ctor
* no need for t at all.. let's just append zeroes directly to bl inside the loop